OneSignalSDK/onesignal/src/main/java/com/onesignal/OSNotificationWorkManager.java

```diff
@@ -84,23 +79,23 @@ private void processNotificationData(Context context, int androidNotificationId,
                     timestamp
             );
 
-            if (OneSignal.notificationProcessingHandler != null)
+            if (OneSignal.notificationProcessingHandler != null) {
                 try {
                     OneSignal.notificationProcessingHandler.notificationProcessing(context, notificationReceived);
                 } catch (Throwable t) {
                     if (!notificationReceived.displayed()) {
```

Comment:
I think this check was ok, this display check inside the if (OneSignal.notificationProcessingHandler != null) and catch, because if the user was able to call display, then the flow continues, we should call quit on the else case